rtl_433_mqtt_autodiscovery/rtl_433_mqtt_hass.py

```diff
@@ -1,676 +0,0 @@
-#!/usr/bin/env python
-# coding=utf-8
-
-# Original source of this script is the RTL_433 Library
-# https://github.com/merbanan/rtl_433/blob/master/examples/rtl_433_mqtt_hass.py
-#
-# It has been modified to incorporate several changes not yet merged into that repo:
-# CHANGELOG:
-#  - https://github.com/pbkhrv/rtl_433/commit/29c04d8b412eb09f4cd1db4b363050d2d72e9065
-#  - https://github.com/pbkhrv/rtl_433/commit/4cf2f952c382d1f49b4bb11d9e0898463e621de0
-#  - https://github.com/merbanan/rtl_433/pull/1665/files (time only)
```

Comment:
This PR still hasn't been merged upstream, so we still need to preserve it: https://github.com/merbanan/rtl_433/pull/1665

Reply:
What do you think about this? https://github.com/grinco/rtl_433-hass-addons/blob/94dc2fcdba942104e4eec196c4c6340b14075e75/rtl_433_mqtt_autodiscovery/Dockerfile#L7
This way you'd have fresh code on every container build/install.

Reply:
https://github.com/pbkhrv/rtl_433/commit/29c04d8b412eb09f4cd1db4b363050d2d72e9065 breaks the moisture sensor discovery (the reason for my PR) - probably one of the reasons it didn't get merged. I think it would make sense to use the upstream source and apply your patches on top of it using a variable in HASS. I've discarded the PR for the time being.

Reply:
See this for example: https://github.com/pbkhrv/rtl_433-hass-addons/pull/50
Once the changes make it upstream patches can be removed from your repo. https://github.com/merbanan/rtl_433/pull/1665/files will need some more work.